backend/app/repository/connection_repository.py

```diff
@@ -0,0 +1,17 @@
+from .base_repository import BaseRepository
+from ..models.connection import ConnectionModel
+
+
+class ConnectionRepository(BaseRepository):
+    def get_collection(self):
+        return self.db['connections']
+
+    def insert(self, connection: ConnectionModel):
+        return super().insert(connection.model_dump())
+
+    def find_all(self) -> list[ConnectionModel] | None:
+        connections = self.find({})
+        if not connections:
+            return None
+        return [ConnectionModel(**connection.get('connection', {})) for connection in
+                connections]
```

Comment:
A jakby tutaj zwracać pustą listę zamiast None? Bo pusta lista w pythonie to też falsy value i ewentualne checki można robić na podstawie tego czy jest pusta czy nie

Reply:
+1